st2reactor/st2reactor/garbage_collector/base.py

```diff
@@ -222,3 +231,15 @@ def _purge_trigger_instances(self):
             LOG.exception('Failed to trigger instances: %s' % (str(e)))
 
         return True
+
+    def _fail_inquiries(self):
+        """Mark Inquiries as "failed" that have exceeded their TTL
```

Comment:
Failed?

Reply:
Good catch, I used to mark the inquiries as failed, but then @Kami suggested I change to "timeout". So this was just a straggler - fixed in 7c5fe270f